common/ferrostar/src/models.rs

```diff
@@ -311,6 +311,11 @@ pub struct RouteStep {
     pub duration: f64,
     /// The name of the road being traveled on (useful for certain UI styles).
     pub road_name: Option<String>,
+    /// A list of exits (name or number), separated by semicolons.
+    ///
+    /// NOTE: This annotation is not in the official spec, but is a common extension used by Mapbox
+    /// and Valhalla.
+    pub exits: Option<String>,
```

Comment:
I think we can/should probably just do the splitting here. For the higher level models, there's no reason to push this work down to the user of the library.

Reply:
done.